src/datasets/datasets.controller.ts

```diff
@@ -2087,7 +2244,7 @@ export class DatasetsController {
       Action.DatasetLogbookRead,
     );
 
-    const proposalId = dataset?.proposalId;
+    const proposalId = (dataset?.proposalIds || [])[0];
```

Comment:
It means the sequence of proposalIds is crucial to retrieve correct logbook messages. 
Do we have control over arbitrary proposalIds update with dataset PATCH or PUT methods?

Reply:
For now, there is only one proposal per dataset, so the problem does not exists.
When we refactor the dataset endpoints, we will brainstorm and think about what to do.

Reply:
...but yes the order of proposals will matter.